ghget

```diff
@@ -8,7 +8,14 @@ usage() {
 	exit 2
 }
 
+version() { 
+	echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'
```

Comment:
What's `$4` here?

Reply:
TBH I get this function straight forward from other use case I had for different semantic versioning.
The correct implementation is `echo "$@" | awk -F. '{ printf("%d%03d%03d\n", $1,$2,$3); }'` or as you kindly proposed here.